newIDE/app/src/ExpressionAutocompletion/index.js

```diff
@@ -513,32 +521,126 @@ type InsertedAutocompletion = {|
   addDot?: ?boolean,
   addParameterSeparator?: ?boolean,
   addNamespaceSeparator?: ?boolean,
+  shouldConvertToString?: ?boolean,
 |};
 
 type ExpressionAndCaretLocation = {|
   expression: string,
   caretLocation: number,
 |};
 
+/**
+ * Returns the position where the last expression call starts,
+ * so that we know where to insert the ToString call.
+ * For example:
+ * '"HelloWorld" + Object.Behavior::' should return 15.
+ */
+const findCompletedExpressionStartPosition = (expression: string) => {
```

Comment:
Let's talk about nodes here:

```suggestion
const findLastNodeStartPosition = (expression: string) => {
```